printer_mods/BladeScraper-Designs/Horizontal-Spool-Holder/readme.md

```diff
@@ -0,0 +1,16 @@
+Based on the stock spool holder, this mod allows you to mount the spool holder to the horizontal top extrusions, rather than the vertical ones, to allow you to choose between more than just the front or back of the printer.  It's sized to fit a KVP 5lb spool or similar, with a 100mm wide area for the spool to sit in.  Just like the stock holder, it uses 4mm PTFE tubing to give the spool a nice smooth surface to ride on.
+
+![alt text](https://github.com/BladeScraper-Designs/VoronUsers/blob/Horizontal-Spool-Holder/printer_mods/BladeScraper-Designs/Horizontal-Spool-Holder/img/Holder.png?raw=true)
+
+BOM: <br>
+2 x M3 T-Nuts <br>
+1 x M3x12 SHCS <br>
+1 x M3x8 SHCS <br>
+1 x Length of PTFE Tubing (Approx 110mm)<br>
+
+To install, first put two M3 T-Nuts into the horizontal extrusion, one on the top and one on the side.  Then, put the M3x12 SHCS through the top hole of the holder, and screw it down loosely into the T-Nut.  Then, after making sure the side hole is aligned, screw the M3x8 SHCS into the other T-Nut.  This screw is technically not required, but does improve strength, as well as keep the holder pointed straight out.  Without this screw, the holder will tilt up slightly without the weight of a spool. Far from a deal breaker, but I would generally suggest using both screws.
+![alt text](https://github.com/BladeScraper-Designs/VoronUsers/blob/Horizontal-Spool-Holder/printer_mods/BladeScraper-Designs/Horizontal-Spool-Holder/img/Holder%201.png?raw=true)
```

Comment:
please change all images to relative links as above.

Reply:
done